data/utils/assembleLsstChannels.py

```diff
@@ -36,16 +36,6 @@
 OutFileName = "image.fits"
```

Comment:
And I think the commit message needs more explanation.  What are you trying to achieve?